tests/phpunit/class-wc-stripe-api-sdk-test.php

```
@@ -0,0 +1,357 @@
+
+	/**
+	 * Create a mock StripeObject whose jsonSerialize() returns $data.
+	 *
+	 * @param mixed $data
+	 * @return \Stripe\StripeObject|\PHPUnit\Framework\MockObject\MockObject
+	 */
+	private function make_stripe_response( $data ) {
+		$mock = $this->createMock( \Stripe\StripeObject::class );
+		$mock->method( 'jsonSerialize' )->willReturn( $data );
+		return $mock;
+	}
```

PHPMD (2.15.0)

[error] The method make_stripe_response is not named in camelCase. (undefined)

(CamelCaseMethodName)